contract/src/main/java/com/ritense/valtimo/contract/database/MysqlQueryDialectHelper.java

```diff
@@ -48,29 +48,43 @@ public <T> Expression<T> getJsonValueExpression(CriteriaBuilder cb, Path column,
 
     @Override
     public Predicate getJsonValueExistsExpression(CriteriaBuilder cb, Path column, String value) {
+        Expression<?> searchColumn = column;
+        Expression<?> searchValue = cb.literal("%" + value.trim() + "%");
+        if (column.getJavaType() == String.class) {
+            searchColumn = cb.function(LOWER_CASE_FUNCTION, String.class, searchColumn);
+            searchValue = cb.function(LOWER_CASE_FUNCTION, String.class, searchValue);
```

Comment:
Mysql `lower()` fails on a column with type 'json'.

Don't know why it didn't fail before